nautobot_firewall_models/utils/capirca.py

```diff
@@ -33,11 +33,17 @@ def _slugify(value):
     dashes to single dashes. Remove characters that aren't alphanumerics,
     underscores, or hyphens. Convert to lowercase. Also strip leading and
     trailing whitespace, dashes, and underscores.
+    Also adds a trailing underscore to names starting with an number to circumvent a bug in the Capirca/Aerleon parser.
```

Comment:
Is this a documented bug? Can we add a link to it here?

Reply:
They require a word at those places and a word cannot start with a number by definiton:
https://aerleon.readthedocs.io/en/latest/reference/yaml_reference/#value-types